tools/online_editor/src/properties_client.ts

```diff
@@ -45,6 +45,23 @@ export async function change_property(
     return new Promise((accept) => accept({ diagnostics: [] }));
 }
 
+export async function remove_property(
```

Comment:
Sometimes this is called remove_property in the code and sometimes removeBinding. I think it's clearly the latter, I suggest to rename the function etc.